spring-cloud-stream/src/main/java/org/springframework/cloud/stream/aggregate/AggregateApplicationBuilder.java

```diff
@@ -164,7 +125,7 @@ public ProcessorConfigurer via(Class<?> processor) {
 
 	}
 
-	private abstract class AppConfigurer {
+	public abstract class AppConfigurer<T extends AppConfigurer<T>> {
```

Comment:
With just the `T` you won't be able to chain methods like in the `embed()` method.
Just try it locally :smile: